src/main/java/com/j256/ormlite/misc/functional/Supplier.java

```diff
@@ -0,0 +1,18 @@
+package com.j256.ormlite.misc.functional;
+
+/**
+ * Represents a supplier of results.
+ *
+ * There is no requirement that a new or distinct result be returned each time the supplier is invoked.
+ *
+ * This is a functional interface intended to support lambda expressions in ormlite whose functional method is get().
+ */
+public interface Supplier<T> {
+
+    /**
+     * Gets a result.
```

Comment:
With these sorts of javadocs, there should just be the body and not a `@return`.  You can take a look at the javadoc output to see what I mean.

Please change to be: Create and return an entity.  Used by Dao. createIfNotExists().

Reply:
I used documentation from java. Why docs must mention concrete usage? It is not an just entity, `Supplier` can provide any value.

Reply:
Makes sense I guess.  Just trying to be more descriptive but I guess the point is to make it non-specific.